scripts/stability-cluster/env_creation.sh

```diff
@@ -0,0 +1,19 @@
+#! /bin/bash
+## Must already have miniconda installed!
+export CONDA_ENV_PATH=/fsx/proj-chemnlp/conda/env/chemnlp-standard
+export PYTHON_VER=3.8
+
+# Create Python environment through conda
+conda create --prefix ${CONDA_ENV_PATH} python=${PYTHON_VER} -y
+conda activate ${CONDA_ENV_PATH}
+
+# Python requirements
+## cd into your directory inside of proj-chemnlp
+cd /fsx/proj-chemnlp/${USER}
```

Comment:
This is relying on the directory names being the same as the user name, which they currently aren't ...

Reply:
ye, I think that might be a safer way going forward to help track back who owns which folders?

Reply:
However, I've also added a positional argument now so that users can specify where the environment should be created for each experiment run. It will still use the cloned copy of the repository in that user's personal directory as I feel that offers good flexibility for running jobs with different branches, etc.